### ISSUE TITLE
 Magento Wizard can’t default content new file

### ISSUE THREAD

**Issue by SmartLJM** [viperet/vscode-magento-wizard#29]
create new extension can works
![1](https://user-images.githubusercontent.com/42052948/127417231-37e6cf4d-2c02-46ae-b458-7ff341a68dd6.jpg)
![image](https://user-images.githubusercontent.com/42052948/127417370-37974907-9474-4d79-b2f7-f871076051d0.png)
but Magento Wizard can’t default content when i create new file,like:
![image](https://user-images.githubusercontent.com/42052948/127417540-0c7ae1a0-afe3-433e-b7cf-88c9b2c0b657.png)
![image](https://user-images.githubusercontent.com/42052948/127417568-d21cea78-ddae-4271-bc98-c7b12917f603.png)
it's empty!
this is my setting
![image](https://user-images.githubusercontent.com/42052948/127417650-687c1058-3cd5-4af2-93f3-d9c6b0862da7.png)
so,what's the problem?

**Comment by viperet** [comment]
@SmartLJM can you open 'Output' tab, switch to 'MagentoWizard' channel, and post log info from it here (after you try to create a block, or invoke some MagentoWizard command from F1 menu). It should look like:

```
PHP 7.0.33 (cli) (built: Aug 14 2019 16:33:54) ( NTS )
Found Magento root at /Users/viperet/Documents/Work/Magento
 - Modules: 151
 - Themes: 11
 - Blocks: 3344
Running '/Applications/MAMP/bin/php/php7.0.33/bin/php bin/magento --no-ansi' in /Users/viperet/Documents/Work/Magento
```

**Comment by SmartLJM** [comment]
![image](https://user-images.githubusercontent.com/42052948/127582449-f32b8862-c9db-42de-8158-70ab5c28eb81.png)
well,does it can't find registration file?that's odd,should i do something special?

**Comment by jcellon** [comment]
Same issue here, nothing defaults or any boilerplate code when creating a class.
Was this issue fixed?

**Comment by viperet** [comment]
@jcellon can you check debug output? Open 'Output' tab, switch to 'MagentoWizard' channel, and post log info from it here (after you try to create a block, or invoke some MagentoWizard command from F1 menu). Usually, that means you don't have PHP executable available, it is used to get info about Magento modules and themes from registration.php files. But for @SmartLJM it is something else, I can't yet reproduce this problem.

**Comment by jcellon** [comment]
@viperet Thanks for the follow up. I did your instructions like: switch to the 'output tab' and when I do `F1` and try to `inject dependency` from the MagentoWizard nothing comes up. But looking into the Runtime Status I get:
 `File not in the workspace (not saved yet?)` I could move this comment to a new issue if that works

**Comment by viperet** [comment]
@jcellon  yes please create a separate issue. Try to create a file, for example etc/di.xml in some module and post log from the output tab in that issue.
